views/address.tmpl

```diff
@@ -23,6 +23,10 @@
                 <div class="row">
                     <a href="javascript:hideAddressQRCode();" id="address-qrcode" class="col opacity-transition d-block" style="opacity: 0"></a>
                 </div>
+                {{if .IsTechnicalAddress}}
+                  <br/>
+                  This address is special. Please update this message once you know why it is special.
```

Comment:
A message here is OK (but not this message ofc), but the main point of issue #358 is to avoid the pointless and expensive DB query.